test/unittest/capability/caldera/caldera_test.go

```diff
@@ -49,6 +49,17 @@ func TestExecuteB64(t *testing.T) {
 	assert.Equal(t, cacao.NewVariables(), results)
 }
 
+func TestParseYamlAbility(t *testing.T) {
+	resultingAbility := caldera.ParseYamlAbility([]byte("ability_id: 9a30740d-3aa8-4c23-8efa-d51215e8a5b9"))
+	assert.Equal(t, resultingAbility.AbilityID, "9a30740d-3aa8-4c23-8efa-d51215e8a5b9")
+}
+
+func TestParseYamlAbilityWithException(t *testing.T) {
+	// This should not crash, just produce an empty Ability
```

Comment:
It is debatable if it is desirable that `ParseYamlAbility` actually errors out silently, we probably want to return an `err` there :)